src/main/java/com/contact/application/controller/DeleteController.java

```diff
@@ -0,0 +1,22 @@
+package com.contact.application.controller;
+
+import com.contact.application.model.Contact;
+import com.contact.application.service.ContactDao;
+import lombok.AllArgsConstructor;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import org.springframework.stereotype.Service;
+
+import java.util.Optional;
+
+@Service
+@Component
```

Comment:
I believe Service and Component are equivalent. Only one is required

Reply:
Ok, will remove